echo-core/src/main/java/com/netflix/spinnaker/echo/notification/AbstractEventNotificationAgent.java

```diff
@@ -164,7 +164,7 @@ public void processEvent(Event event) {
     }
 
     List<Map<String, Object>> notifications =
-        (List<Map<String, Object>>) context.get("notifications");
+        (List<Map<String, Object>>) context.getOrDefault("notifications", Collections.emptyList());
```

Comment:
oops, this was from an exception I noticed when somehow `notifications` is inexplicably `null`...  Since it's a pretty benign change, will just leave it as part of this PR unless anyone objects